cabot/metricsapp/tests/test_elasticsearch.py

```diff
@@ -197,6 +201,27 @@ def test_terms_aggregation(self):
         self.assertEquals(result.error, 'maroon.max: 18.3 < 18.0')
         self.assertEqual(self.es_check.importance, Service.CRITICAL_STATUS)
 
+    @patch('cabot.metricsapp.models.elastic.MultiSearch.execute', fake_es_filters_aggregation)
+    @patch('time.time', mock_time)
+    def test_filters_aggregation(self):
+        series = self.es_check.get_series()
+        self.assertFalse(series['error'])
+        self.assertEqual(series['raw'], get_json_file('es_filters_aggregation.json'))
+        data = series['data']
+        self.assertEqual(len(data), 2)
+        data = sorted(data, key=lambda d: d['series'])
+        import logging
+        logger = logging.getLogger(__name__)
+        logger.error("zaa\n\n")
+        logger.error(data)
+        logger.error("\n\n")
+
+        self.assertEqual(data[0]['series'], 'status_code:301.value_count: ')
```

Comment:
something strange with how its parsed - the key itself has that. was reluctant to just slice off the last 3 characters...